src/components/node-list/node-list-row.js

```diff
@@ -77,7 +78,10 @@ const NodeListRow = memo(
         onMouseEnter={visible ? onMouseEnter : null}
         onMouseLeave={visible ? onMouseLeave : null}>
         <TextButton
-          className="pipeline-nodelist__row__text"
+          className={classnames('pipeline-nodelist__row__text', {
+            'pipeline-nodelist__row__text--tree': rowType === 'tree',
+            'pipeline-nodelist__row__text--filter': rowType !== 'tree',
```

Comment:
Looks like this may not extend further than these two types - is there any way this could be made generic e.g. `pipeline-nodelist__row__text--${rowType}`?

Reply:
I've incorporated your suggestion here in [this commit ](https://github.com/quantumblacklabs/kedro-viz/pull/479/commits/b219503b82486c1216ca2803671562aa91776232)